src/ai/backend/common/bgtask/types.py

```diff
@@ -0,0 +1,12 @@
+import enum
+
+
+class TaskStatus(enum.StrEnum):
+    STARTED = "bgtask_started"
+    DONE = "bgtask_done"
+    CANCELLED = "bgtask_cancelled"
+    FAILED = "bgtask_failed"
+    PARTIAL_SUCCESS = "bgtask_partial_success"
+
+    def finished(self) -> bool:
+        return self in {self.DONE, self.CANCELLED, self.FAILED, self.PARTIAL_SUCCESS}
```

Comment:
Maybe it might be better
```suggestions
return self not in {self.STARTED}
```